src/components/Primer/Primer.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+import { Primer as PrimerSDK } from '@primer-io/checkout-web';
+
+const clientToken = 'abc';
+
+const options = {
+  container: '#msd__primerWrapper'
+};
+
+const Primer = () => {
+  const createDropIn = async () => {
```

Comment:
please move this inside the `useEffect`'s callback. right now the function will be recreated on every render, versus being created only once during the component's mount